bokeh/models/widgets/inputs.py

```diff
@@ -242,42 +244,6 @@ class MultiSelect(InputWidget):
     """)
 
 
-class MultiChoice(InputWidget):
-    ''' MultiChoice widget.
-
-    '''
-
-    options = List(Either(String, Tuple(String, String)), help="""
-    Available selection options. Options may be provided either as a list of
-    possible string values, or as a list of tuples, each of the form
-    ``(value, label)``. In the latter case, the visible widget text for each
-    value will be corresponding given label.
-    """)
-
-    value = List(String, help="""
-    Initial or selected values.
-    """)
-
-    delete_button = Bool(default=True, help="""
-    Whether to add a button to remove a selected option.
-    """)
-
-    max_items = Int(default=None, help="""
-    The maximum number of items that can be selected.
-    """)
-
-    option_limit = Int(default=None, help="""
-    The number of choices that will be rendered in the dropdown.
-    """)
-
-    placeholder = String(default=None, help="""
-    A string that is displayed if not item is added.
-    """)
-
-    solid = Bool(default=True, help="""
-    Specify whether the choices should be solidly filled.""")
-
-
```

Comment:
Any idea what's going on here? This was added in  recent other PR and should not be removed. :) If you are not comfortable with git command line (merging, rebasing master etc) then it might be simpler to pull the latest master in to your fork, then make a new branch/PR off that.

Reply:
The first commit needs to be removed to fix this, which is the simplest way, also preserving all review comments.

Reply:
This passage is non-existent in my fork.
Do I understand your @bryevdv proposal correct, to pull the current master before I start doing my code revision? 
And than trigger a new PR??

Reply:
Created a new branch and a new PR #9732 based on the 
correct master.